Cloudinary/Classes/Core/CLDConfiguration.swift

```diff
@@ -26,16 +26,16 @@ import Foundation
 
 /**
  The CLDConfiguration class holds the configuration parameters to be used by the `CLDCloudinary` instance.
-*/
+ */
```

Comment:
It's very problematic to commit formatting/style along with actual changes - very hard to track.